examples/distro/gradle/instrumentation.gradle

```diff
@@ -40,13 +40,11 @@ shadowJar {
 tasks.withType(Test).configureEach {
   jvmArgs "-Dotel.javaagent.debug=true"
   jvmArgs "-javaagent:${configurations.testAgent.files.first().absolutePath}"
-  jvmArgs "-Dotel.initializer.jar=${shadowJar.archiveFile.get().asFile.absolutePath}"
-  jvmArgs "-Dinternal.testing.disable.global.library.ignores=true"
-  // always run with the thread propagation debugger to help track down sporadic test failures
-  jvmArgs "-Dotel.threadPropagationDebugger=true"
-  jvmArgs "-Dotel.internal.failOnContextLeak=true"
+  jvmArgs "-Dotel.javaagent.experimental.initializer.jar=${shadowJar.archiveFile.get().asFile.absolutePath}"
+  jvmArgs "-Dotel.javaagent.testing.additional-library-ignores.enabled=true"
```

Comment:
This looks like incorrect change. Old option was "xxx.disable=true", new one is "xxx.enabled=true".

Reply:
oh good catch!